components/photo/Modal.vue

```diff
@@ -47,12 +47,12 @@ useEventListener('keydown', (e) => {
       aria-hidden="true"
       max-w-full max-h-full object-contain
     />
-    <div absolute left-0 top="1/2">
+    <div absolute left-0 top="calc(1/2-58px)">
```

Comment:
```suggestion
    <div absolute left-0 top="1/2" translate="y--1/2">
```

Maybe this would be more robust

Reply:
Absolutely!